tests/unit/render/test_render.cc

```diff
@@ -65,44 +71,157 @@ class RenderTest :public ::testing::TestWithParam<std::string> {
 
   void SetUp() override {
     // Make the output directory for these tests
-    std::filesystem::create_directory(fmt::format("{}/output", SRC_DIR));
-    std::filesystem::create_directory(fmt::format("{}/output/tests", SRC_DIR));
-    std::filesystem::create_directory(fmt::format("{}/output/tests/render", SRC_DIR));
+    // std::filesystem::create_directory(fmt::format("{}/output", SRC_DIR));
+    // std::filesystem::create_directory(fmt::format("{}/output/tests", SRC_DIR));
+  }
```

Comment:
Same here--is the `SetUp` necessary still?

Reply:
method removed